### ISSUE TITLE
Cannot edit "Open new chat rooms in the default container" setting when using a screen reader on Windows

### ISSUE THREAD

**Issue by daniel-montalvo** [miranda-ng/miranda-ng#2023]
When using the latest  Miranda NG and NVDA, cannot change "Open new chatrooms in the default container" status.

Steps to reproduce.
1. Run NVDA screen reader.
2. Go to the Miranda Options 
3. Use the arrow keys on your keyboard to select the "Open new chatrooms in the default container: setting and try to check/uncheck it.

Expected result
NVDA will report current status of that checkbox and will update its status when the space keyboard is pressed.

Current behavior

NVDA will not report the current status of the checkbox and an error sound will play when the space keyboard is pressed to toggle the status.

**Comment by daniel-montalvo** [comment]
@georgehazan Thanks for this. Awesome that you take this into account so quickly! How can I get a version of the product with this working?

Thanks!

**Comment by dartraiden** [comment]
You can switch to development version in updater settings and wait for
update. Dev version receives updates almost everyday.
